utils/api/benefits/afsBenefit.ts

```diff
@@ -128,15 +128,8 @@ export class AfsBenefit extends BaseBenefit {
           detail: this.translations.detail.dependingOnLegal,
         }
       }
-    } else if (this.input.livingCountry.noAgreement) {
-      return {
-        result: ResultKey.INELIGIBLE,
-        reason: ResultReason.SOCIAL_AGREEMENT,
-        detail: this.translations.detail.ineligibleYearsOrCountry,
-      }
```

Comment:
Devil's advocate here, should this condition stay? If we aren't sure it's unreachable is it safer to just leave it in? It's technically correct and a valid reason the calculator should spit out.

Just playing the other side, if you're confident -- we can try this and if it breaks we just fix it later

Reply:
I'm pretty confident it's fine! I spent a little while going through the conditions and as far as I can tell it's totally unreachable. I wanted to get rid of it because the reason detail text is way too long. If it becomes an issue we can handle it better than it was being handled before this PR's changes.